src/System.Net.Http/tests/FunctionalTests/DefaultCredentialsTest.cs

```diff
@@ -292,4 +292,11 @@ public bool IsBypassed(Uri host)
             }
         }        
     }
+
+    public sealed class ManagedHandler_DefaultCredentialsTest : DefaultCredentialsTest, IDisposable
+    {
+        public ManagedHandler_DefaultCredentialsTest(ITestOutputHelper output) : base(output) => ManagedHandlerTestHelpers.SetEnvVar();
+        public void Dispose() => ManagedHandlerTestHelpers.RemoveEnvVar();
+    }
```

Comment:
Why did you move this to this file? We should be consistent, and currently all of the derived ManagedHandler tests are in the other file. This is now inconsistent.